pkg/runctl/config.go

```diff
@@ -43,10 +47,21 @@ func loadPipelineRunsConfig(clientFactory k8s.ClientFactory) (*pipelineRunsConfi
 		return nil, err
 	}
 
+	networkMap, err := configMapIfce.Get(networkPoliciesConfigMapName, metav1.GetOptions{})
+	// Do we need this only for testing?
+	if k8serrors.IsNotFound(err) {
+		return &pipelineRunsConfigStruct{}, nil
+	}
```

Comment:
This is wrong. If the configmap does not exist, treat it as if it was empty (no data entries), but do not return an empty config struct.